recipe/conda_forge_ci_setup/inspect_artifacts.py

```diff
@@ -1,15 +1,29 @@
 from pathlib import Path
+import os
 
 import click
 import conda_build.config
 from conda_package_handling.api import list_contents
 
-from .utils import built_distributions, compute_sha256sum, human_readable_bytes
+from .utils import (
+    built_distributions,
+    compute_sha256sum,
+    get_built_distribution_names_and_subdirs,
+    human_readable_bytes,
+)
 
 
 @click.command()
 def main():
```

Comment:
for eg: `variant` and `recipe_dir` should be added as `click` options and passed through

Reply:
xref: https://github.com/conda-forge/conda-forge-ci-setup-feedstock/issues/345